net_ipc/src/errors.rs

```diff
@@ -0,0 +1,38 @@
+//! This module holds net_ipc custom error types.
+
+use failure;
+use std;
+
+/// net_ipc-specific error types
+#[derive(Debug, Clone, Fail)]
+pub enum IpcError {
+    /// Translate an Option<_> unwrap into a Result::Err
+    #[fail(display = "NoneError")]
+    NoneError,
```

Comment:
Can we change this to a better error name that is more explicit on what went wrong like "MissingData" ?

Reply:
:+1: done